OTAnalytics/plugin_number_of_tracks_to_be_validated/svz/number_of_tracks_to_be_validated.py

```diff
@@ -0,0 +1,62 @@
+from pandas import DataFrame
+
+from OTAnalytics.application.logger import logger
+from OTAnalytics.application.use_cases.highlight_intersections import (
+    TracksAssignedToAllFlows,
+)
+from OTAnalytics.application.use_cases.number_of_tracks_to_be_validated import (
+    NumberOfTracksToBeValidated,
+)
+from OTAnalytics.domain.track import OCCURRENCE, TRACK_CLASSIFICATION, TRACK_ID
+from OTAnalytics.plugin_number_of_tracks_to_be_validated.calculation_strategy import (
+    DETECTION_RATE,
+    DetectionRateCalculationStrategy,
+)
+from OTAnalytics.plugin_number_of_tracks_to_be_validated.metric_rates_builder import (
+    MetricRatesBuilder,
+)
+from OTAnalytics.plugin_number_of_tracks_to_be_validated.tracks_as_dataframe_provider import (  # noqa
+    TracksAsDataFrameProvider,
+)
+
+
+class SvzNumberOfTracksToBeValidated(NumberOfTracksToBeValidated):
+    def __init__(
+        self,
+        tracks_provider: TracksAsDataFrameProvider,
+        tracks_assigned_to_all_flows: TracksAssignedToAllFlows,
+        detection_rate_strategy: DetectionRateCalculationStrategy,
+        metric_rates_builder: MetricRatesBuilder,
+    ) -> None:
+        self._tracks_provider = tracks_provider
+        self._tracks_assigned_to_all_flows = tracks_assigned_to_all_flows
+        self._detection_rate_strategy = detection_rate_strategy
+        self._rates_builder = metric_rates_builder
+
+    def calculate(self) -> int:
+        tracks_df = self._tracks_provider.provide()
+        if tracks_df is None:
+            return 0
+        data = self._get_tracks_assigned_to_flow(tracks_df.reset_index())
+        rate = self._detection_rate_strategy.calculate(data).reset_index()
+        metric_rates = self._rates_builder.build_as_dataframe()
+        merged = rate.merge(metric_rates, how="left", on=TRACK_CLASSIFICATION)
+        merged[DETECTION_RATE] = merged[DETECTION_RATE].round(2)
+        number_of_tracks = len(merged)
+        filter_column = "manual_classification"
+        merged[filter_column] = merged[DETECTION_RATE] < merged["Gruppe A3"]
```

Comment:
`Gruppe A3` is also used in `metric_rates.py` please use a single source for the name. We might also rename it to "svz-rate"